compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1324,20 +1339,30 @@ SmallVector<int64_t, 4> AttentionOp::getStaticLoopRanges() {
 
 SmallVector<AffineMap> AttentionOp::getIndexingMapsForOperands() {
   auto maps = getIndexingMapsArray();
-  return SmallVector<AffineMap>(maps.begin(),
-                                maps.begin() + getNumDpsInputs() - 1);
+  return SmallVector<AffineMap>(maps.begin(), maps.begin() + getNumDpsInputs());
```

Comment:
IIRC this returns a vector, so we should be able to resize it without constructing a new one.

Reply:
What exactly do you mean?

Reply:
IE truncate the size to only keep the elements we need

Reply:
Ahh got it